content/docs/command-reference/plots/index.md

```diff
@@ -1,20 +1,20 @@
 # plots
 
-Contains commands to visualize _plot metrics_ in structured files (JSON, CSV, or
-TSV): [show](/doc/command-reference/plots/show),
-[diff](/doc/command-reference/plots/diff).
+A set of commands to visualize and compare _plot metrics_ in structured files
+(JSON, CSV, or TSV): [show](/doc/command-reference/plots/show),
```

Comment:
Also YAML.

Reply:
Hmmm I thought this was removed? Kind of lost with some 1.0 updates, sorry. Can you point to the core PR that adds this? It's probably better to follow up on the doc updates there for now.

Reply:
@jorgeorpinel can we try to run it? or check the code in core repo?

Reply:
It let's me add a YAML file with `dvc run --plots` (then again it seems I can add any file with any format or file extension) but I'm getting `ERROR: unexpected error - 'str' object has no attribute 'keys'` with that I think is a valid yaml file:
```yaml
---
parabola:
 - y: 1
 - y: 2
 - y: 4
 - y: 8
```

I think it would be best to follow up on this in the PR that introduces YAML plots and/or extract it form this PR for now.

Reply:
> getting ERROR: unexpected error

That's when I run `dvc plots show`

Reply:
> follow up on this in the PR that introduces YAML plots

Meaning in the core repo. I wasn't able to easily find it BTW. Also, examples should be written in the cmd refs.

Reply:
As far as I remember, YAML was not implemented for plots. And this is an issue. We should include YAML support for plots.

Reply:
Created https://github.com/iterative/dvc/issues/3970

Reply:
Thanks!

Reply:
@jorgeorpinel Yaml will be supported after https://github.com/iterative/dvc/pull/3992